modules/idlgen/plugin/src/main/scala/IdlGenPlugin.scala

```diff
@@ -79,8 +83,9 @@ object IdlGenPlugin extends AutoPlugin {
     idlGenSourceDir := (Compile / sourceDirectory).value,
     idlGenTargetDir := (Compile / resourceManaged).value,
     srcGenSerializationType := "Avro",
+    srcGenSourceFromJarsDir := idlGenTargetDir.value / idlType.value,
+    srcGenSourceDir := Seq((Compile / resourceDirectory).value, srcGenSourceFromJarsDir.value),
```

Comment:
Would making this a collection require changing the corresponding `srcGen` arg? (currently `srcGenSourceDir.value.allPaths.get.toSet`)